util/msg_service.go

```diff
@@ -0,0 +1,45 @@
+package util
```

Comment:
Let's rename the package and directory to `/client`.

Reply:
ack